lib/pure/asyncdispatch.nim

```diff
@@ -246,6 +246,30 @@ type
     timers*: HeapQueue[tuple[finishAt: MonoTime, fut: Future[void]]]
     callbacks*: Deque[proc () {.gcsafe.}]
 
+  PStatus = enum
+    PENDING, CANCELED, RUNNING, FINISHED
```

Comment:
We don't SCREAM constants in Nim, constants are most benign. Better to focus the attention of the reader to something more interesting.

Reply:
Ok. I'll patch it, soon.